app/helpers/administrate/application_helper.rb

```diff
@@ -19,6 +19,16 @@ def display_resource_name(resource_name)
         )
     end
 
+    def sort_order(order)
+      if order == "asc"
+        "ascending"
+      elsif order == "desc"
+        "descending"
+      else
+        "none"
+      end
```

Comment:
I updated the helper to use a case statement in 33a0da4233b275f2fda1d48ef9422d65e14fb758. It can be squashed before merging.